docs/docs-beta/docs/api/index.md

```diff
@@ -0,0 +1,8 @@
+---
+sidebar_class_name: hidden
+title: API Docs
+---
+
+import DocCardList from '@theme/DocCardList';
+
+<DocCardList />
```

Comment:
this means we need to make sure `make clean` doesn't wipe this file

Reply:
Good call, I've excluded it in https://github.com/dagster-io/dagster/pull/24352/commits/2a0a82e43c588baa8a687ca9f053c1687f28f01d